client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -382,11 +383,12 @@ const ElectionResultsCard: React.FC<IProps> = ({
             <tr>
               <td>
                 <Button
+                  aria-label={isMobileWidth ? 'Add Candidate' : undefined}
                   disabled={!editable}
                   icon="plus"
                   onClick={() => addCandidate(constructNewCandidate())}
                 >
-                  Add Candidate
+                  {(isTabletWidth || isDesktopWidth) && 'Add Candidate'}
                 </Button>
```

Comment:
As noted in the PR description, there's still room for improvement. Remaining issues include:

- Form elements hopping around when error strings exceed two lines
- Labels wrapping on really small screens
- The "Clear" confirmation modal not being full screen

But this PR covers the basics